lib/elements/current_day.dart

```diff
@@ -0,0 +1,193 @@
+import 'package:flutter/material.dart';
+
+class CurrentDay extends StatefulWidget {
```

Comment:
Do you use any setState, if not please make widget stateless

Reply:
Fixed, Thanks